docs/traffic-policy/concepts/phase-rules.mdx

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 2
+title: Phase Rules
+pagination_previous: traffic-policy/concepts/phases
+pagination_next: traffic-policy/concepts/expressions
+---
+
+# Phase Rules
+
+Each Phase is made up of Phase Rules. Phase Rules allow you to define how traffic is filtered and processed within a phase. To do this each rule consists of two key parts: **expressions** and **actions**.
+
+Here is an example:
+
+```yaml
+on_tcp_connect:
+  - expressions:
+      - conn.client_ip == '192.168.1.200'
+    actions:
+      - type: deny
+```
+
+## Expressions
+
+Expressions are conditions written in [Common Expression Language (CEL)](https://github.com/google/cel-spec) that can be used to evaluate specific traffic attributes, such as the client IP, request URL, or HTTP method. These conditions determine whether a rule applies to a given traffic flow. For instance, an expression like `conn.client_ip == '192.168.1.200'` targets requests from a specific IP address.
```

Comment:
```suggestion
Expressions are written in [Common Expression Language (CEL)](https://github.com/google/cel-spec) and have access to all traffic attributes, such as the client IP, request URL, or HTTP method. These expressions determine whether the actions defined in a rule apply to the traffic flow. For instance, an expression like `conn.client_ip == '192.168.1.200'` would match traffic from a specific IP address.
```

Reply:
This isn't exactly true, will adjust wording and using conditions here since we eventually want to move over to that.